### ISSUE TITLE
Accessibility: Status badges don't have enough contrast.

### ISSUE THREAD

**Issue by GuySartorelli** [silverstripe/silverstripe-versioned-admin#414]
[WCAG 2.2 AA 1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)

The "Draft" and "Modified" status badges don't have enough contrast.

It's not clear from the report if these are the _only_ status badges that fail contrast, nor whether it's for just the regular badge or also the `badge--breadcrumbs` variants, so we should check them all.

## Acceptance criteria

- [x] All status flag styles meet accessibility contrast standards

## PRs

- https://github.com/silverstripe/silverstripe-admin/pull/2084
- https://github.com/silverstripe/silverstripe-versioned-admin/pull/448

**Comment by cassiezoest** [comment]
### Draft/modified badges

Update the text/border colour for these to #C73C00 to meet the contrast requirement.

This is optional, but personally I feel like the background colour is too light... if we wanted to darken it a touch and keep a compliant contrast ratio:

Background: #FFEDE0
Text/border: #C23A00

### Custom badges

Default colours are all good contrast wise.

**Comment by emteknetnz** [comment]
Linked PRs have been merged, they will be released as part of CMS 6.2
